File: engine/model/src/main/scala/net/server/channel/packet/cash/use/UseIncubatorPacket.scala
package net.server.channel.packet.cash.use

class UseIncubatorPacket(private var _position: Short, private var _itemId: Int, private var _itemType: Byte, private var _slot: Short) extends AbstractUseCashItemPacket(_position, _itemId) {
  def itemType: Byte = _itemType

  def slot: Int = _slot
}

File: engine/engine-model/src/main/java/net/server/channel/packet/cash/use/UseIncubatorPacket.java
package net.server.channel.packet.cash.use;

public class UseIncubatorPacket extends AbstractUseCashItemPacket {
   private final byte itemType;

   private final int slot;

   public UseIncubatorPacket(short position, int itemId, byte itemType, int slot) {
      super(position, itemId);
      this.itemType = itemType;
      this.slot = slot;
   }

   public byte itemType() {
      return itemType;
   }

   public int slot() {
      return slot;
   }
}
